game/mushcnf.dst

```diff
@@ -541,6 +541,11 @@ full_html_file txt/full.html
 # OPTIONAL
 # who_html_file
 
+# Static HTML landing page to show when someone accesses the MUSH
+# port using a web browser and no http_handler is defined.
+# OPTIONAL
+# index_html_file txt/index.html
```

Comment:
- What is the expected documentation if you have both a mudurl and this? I feel documentation could be improved here.
- The idea of serving up a static page instead of using an http_handler seems an error in logic as-is. If anything, this could serve as a 'default' page when http_handler fails, or something of the like.

Reply:
This is the same behavior as before. When there is no http_handler to handle requests it builds a hardcoded page and dumps that back on HTTP requests. The only thing that mudurl determines is whether or not that hardcoded page contains a link to mudurl. In this PR we we just dump a static index.html if it's defined, otherwise build the hardcoded page as we normally would.